usdf/onboarding.rst

```diff
@@ -0,0 +1,103 @@
+############################
+SLAC Onboarding Procedure
+############################
+
+Overview
+=============================
+
+The SLAC onboarding procedure involves the following steps:
+
+- join the SLAC HEP users organization, SLUO
+   - fill out the SLAC Users Organization (SLUO) `form <https://oraweb4.slac.stanford.edu/apex/epnprod/f?p=134:1::::::>`__
+   - fill out the SLAC Site Access Portal `form <https://vue.slac.stanford.edu/content/slac-site-access-portal>`__
+- a SLAC ServiceNow ticket is submitted to IT to request the account
+- IT creates the account and sends a link to reset the passwords
+  (note: this will result in 2 accounts at present: a Windows account
+  used for authentication to SDF, and a unix account) 
+- do the SLAC Cyber training
+
+Procedure Details
+=============================
+
+To obtain a SLAC SID number and SLAC account, you first need to become a SLAC User. Please follow the below steps and complete the registration form.  
+
+
+1)	New users are required to complete the SLUO registration form using the below `form <https://oraweb4.slac.stanford.edu/apex/epnprod/f?p=134:1::::::>`__.
+
+2)	The first page of the SLUO registration form asks for new users’ information (screenshot below). Once you complete the form, click the “Continue” button.
+
+
+.. image:: /_static/usdf/dev_guide/SLUO_New_User_Form.png
+
+Notes:
+ - classification: select Contractor or Consultant if you have that direct relationship to SLAC, otherwise use your home institution classification.
+ - work area: most answers for Rubin Ops will be "Physics Research" or "Computing"
+ - Experiment: select LSST
+ - SLAC Spokesperson/Sponsor/Supervisor: Select Richard Dubois.
+ - Time at SLAC: this is physically on site. For most people, this is <10%.
```

Comment:
"Time at SLAC" and "Start date" should move to below the second page graphic since they don't appear until page 2.

Reply:
Done